wrangling.Rmd

```diff
@@ -1210,27 +1185,26 @@ lang_summary <- summarize(region_lang,
 ```
 
 From this we see that there are some languages in the data set the no one speaks
-as their primary language at home, as well as that the most commonly spoken
+as their primary language at home. We also see that the most commonly spoken
 primary language at home is spoken by
 `r format(lang_summary$max_most_at_home[1], scientific = FALSE, big.mark = ",")`
 people.
 
 #### Calculating summary statistics when there are `NA`s {-}
 
-When we use base R statistical summary functions 
-(e.g., `max`, `mix`, `mean`, `sum`, etc) with `summarize`, 
-R will return to us `NA` when we apply functions to columns that 
-contain `NA`s in the data frame. 
-Usually that is not what we want to happen, 
-instead we'd usually like R to ignore the `NA`s 
-and calculate the summary statistic using all of the other values, 
-that are not `NA`s, in the column.
-
-How can we do that? 
-In this case of pairing base R statistical summary functions with `summarize`, 
-we can avoid getting R to return `NA`s by adding the argument `na.rm = TRUE` 
-to the base R statistical summary function we are applying. 
-We show an example of this below.
+In data frames in R, the value `NA` is often used to denote missing data. 
+Many of the base R statistical summary functions 
+(e.g., `max`, `mix`, `mean`, `sum`, etc) will return `NA` 
+when applied to columns containing `NA` values. \index{missing data}\index{NA|see{missing data}}
+Usually that is not what we want to happen; 
+instead, we would usually like R to ignore the missing entries
+and calculate the summary statistic using all of the other non-`NA` values
+in the column.
+Fortunately many of these functions provide an argument `na.rm` that lets
+us tell the function what to do when it encounters `NA` values.
+In particular, if we specify `na.rm = TRUE`, the function will ignore
+missing values and return a summary of all the non-missing entries.
+We show an example of this combined with `summarize` below.
 
 First we create a seemingly innocuous `NA` 
 in the first row of the `region_lang` data frame, 
```

Comment:
I think this code adding the NA may be too much at the moment to show, especially since we don't really explain it . I think we should create it under the hood and read it in or explain this further

Reply:
I vote just doing it under the hood.

Reply:
agree, fixed